examples/next/template-hierarchy/example-app/src/wp-templates/single.js

```diff
@@ -0,0 +1,36 @@
+import Layout from "@/components/Layout";
+
+export default function SingleTemplate({ graphqlData }) {
+  const { SinglePostQuery } = graphqlData;
+  return (
+    <Layout>
+      <h2>{SinglePostQuery.response.data.post.title}</h2>
+      <div
+        dangerouslySetInnerHTML={{
+          __html: SinglePostQuery.response.data.post.content,
```

Comment:
```suggestion
      <h2>{SinglePostQuery.data.post.title}</h2>
      <div
        dangerouslySetInnerHTML={{
          __html: SinglePostQuery.data.post.content,
```


A small update to fix the issue below:

<img width="1024" height="573" alt="Image" src="https://github.com/user-attachments/assets/e6fa65d9-1bdc-467b-ad22-c8f14280d08f" />